src/java/bobbybot/Parser.java

```diff
@@ -0,0 +1,93 @@
+package bobbybot;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Parser {
```

Comment:
Perhaps a header comment here for this class could help to explain the functionality of this class?